action.yml

```diff
@@ -73,6 +73,20 @@ runs:
         echo "::set-output name=PACKAGE_VERSION::$(echo $(node -p "require('./package.json').version"))"
       shell: 'bash'
 
+    - name: Get package version
+      id: get-package-version
+      if: ${{ github.event_name == 'pull_request' }}
+      run: |
+        echo "::set-output name=PACKAGE_VERSION::$(echo $(node -p "require('./package.json').version"))"
+      shell: 'bash'
+
+    - name: Get package name
+      id: get-package-name
+      if: ${{ github.event_name == 'pull_request' }}
+      run: |
+        echo "::set-output name=PACKAGE_NAME::$(echo $(node -p "require('./package.json').name"))"
+      shell: 'bash'
```

Comment:
a few comments on this:

- can't this be done in a single step?
- also, since we're setting the same variable earlier, I think there is a chance for refactoring it. plus, are we sure this is the easiest way to grab this information? I mean isn't it just simpler to do it in code than here?

Reply:
The package info is now loaded in code as per suggestion.